src/util/tabTools.ts

```diff
@@ -0,0 +1,14 @@
+// Constants
+import { FAIL } from "../constants/transactions";
+
+export const errorToast = (error: Error, value: string, openToast: (identifier: string, status: string, message: string) => void, defaultToast: () => void) => {
```

Comment:
split this over a few lines so it's easier to see the different parameters please